File: node-service/sort.js
let { client, messages} = require("./lib/grpc");

let call = client.sort(function (error, response) {
    if(error){
        console.log(error)
        return
    }
    console.log("the sorted array is ", response.array[0])
})

function main() {
    process.argv.splice(2).forEach(number => {
        number = parseInt(number, 10)
        let request = new messages.SortRequest()
        request.setNum(number)
        call.write(request)
    })
    call.end();
}

main()

File: go-service/lib/sort.go
package lib


func Sort(numbers []int32) []int32{

	for i := len(numbers); i > 0; i-- {
		
		for j := 1; j < i; j++ {
			if numbers[j-1] > numbers[j] {
				intermediate := numbers[j]
				numbers[j] = numbers[j-1]
				numbers[j-1] = intermediate
			}
		}
	}
	return numbers
}